libpod/network/util/ip.go

```diff
@@ -0,0 +1,115 @@
+package util
+
+import (
+	"math/rand"
+	"net"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+func init() {
+	rand.Seed(time.Now().UnixNano())
```

Comment:
libpod/kube.go also calls rand.Seed(), should only be called once per library instantiation. Not sure where that should happen. @mheon ideas?

Reply:
Why should this be only called once? I fear that the seed gets removed in the other place, e.g. when the code is refactored, and this package ends up without seed.

Reply:
@Luap99 In general, reseeding a software pseudo-random generator reduces the randomness. It also makes "locking" the generated sequence almost impossible. Tests may want to lock the sequence.  This is more a "code smell" then an error.

Reply:
Any reason we shouldn't just write a function that grabs N bits of entropy from `/dev/urandom` and use that instead?

Reply:
Actually I can just use the [crypto/rand](https://pkg.go.dev/crypto/rand) package. This doesn't need seed and works on all platforms.